airo-robots/airo_robots/manipulators/position_manipulator.py

```diff
@@ -205,6 +208,67 @@ def is_tcp_pose_reachable(self, tcp_pose: HomogeneousMatrixType) -> bool:
             return False
         return self._is_joint_configuration_reachable(joint_configuration)
 
+    def execute_trajectory(self, joint_trajectory: SingleArmTrajectory) -> None:
+        """Execute a joint trajectory. This function will interpolate the trajectory and send the commands to the robot.
+
+        This function is implemented according to the notes of https://github.com/airo-ugent/airo-mono/issues/150.
+        Please refer to this issue for design decisions.
+
+        Args:
+            joint_trajectory: the joint trajectory to execute."""
+        self._assert_joint_trajectory_is_executable(joint_trajectory)
+
+        period = 0.005  # Time per servo, approximately. This may be slightly changed because of rounding errors.
```

Comment:
how did you determine this value?

Reply:
I took it from the [cloth competition codebase](https://github.com/Victorlouisdg/cloth-competition/blob/34dcac88c9dd1aaaccbc3fd4b06c86651a70cd0c/cloth-tools/cloth_tools/trajectory_execution.py#L39), I suppose 0.002 would also be a fair choice (500Hz)